.cirrus.yml

```diff
@@ -34,21 +34,21 @@ style_task:
       -DCMAKE_BUILD_TYPE="$BUILD_TYPE" -DCPACK_GENERATOR=TGZ
       -DCPACK_PACKAGE_FILE_NAME="$(cat PACKAGE_NAME)"
       -DCPACK_SET_DESTDIR=ON "$CONFIGURE_FLAGS" -G Ninja
-    - cmake --build build --target all
+    - cmake --build build --target all --parallel
   unit_test_script:
-    - cmake --build build --target test
+    - cmake --build build --target test --parallel
```

Comment:
The targets `test`, `integration`, `install` and `package` do not benefit from the `--parallel` option.